src/models/item.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require('mongoose');
+const isImageUrl = require('is-image-url');
+
+const validator = (url) => isImageUrl(url);
+const imageValidator = [validator, 'Please enter a valid image URL!'];
```

Comment:
Can you double-check if the validate property on mongoose takes an array format or an object format? I think it might be object format.

Reply:
There is an array of objects, but they use it in the example when there is more than one validator
This example is from the docs using the same method that I used: 
```
// make sure every value is equal to "something"
function validator (val) {
  return val == 'something';
}
new Schema({ name: { type: String, validate: validator }});

// with a custom error message

const custom = [validator, 'Uh oh, {PATH} does not equal "something".']
new Schema({ name: { type: String, validate: custom }});
```

Reply:
And thank you @Shrreya for the commit message correction. I make this mistake a lot; I will pay more attention to that from now on 😁

Reply:
Okay I sometimes forget if the format was array or object, but usually mongoose accepts both. Thank you for confirming.